### PR TITLE
Fix dygraph doc at nn.py for v1.5 

### PR DESCRIPTION
**Pull request by junjun315**
fix doc: conv3d, layer norm, bilineartensorproduct, conv2dtranspose, conv3dtranspose,rowconv, groupnorm, spectralnorm, treeconv

hidden api: sequenceConv

### CODE REVIEW

File: python/paddle/fluid/dygraph/nn.py [hunk 1]
```diff
@@ -2168,6 +2211,49 @@ def forward(self, input):
...
+    from future subsequences in a computationally efficient manner to improve
+    unidirectional recurrent neural networks. The row convolution operator is
+    different from the 1D sequence convolution, and is computed as follows:
+
+    Given an input sequence X of length t and input dimension D, and a filter (W) of size context * D.
+
+    More details about row_conv please refer to the design document https://github.com/PaddlePaddle/Paddle/issues/2228#issuecomment-303903645 .
+
+    Args:
+        input (${x_type}): ${x_comment}.
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/nn.py]: don't need `input` here?

File: python/paddle/fluid/dygraph/nn.py [hunk 2]
```diff
@@ -2290,6 +2386,63 @@ def forward(self, input):
...
+
+        \sigma(\mathbf{W}) = \mathbf{u}^{T} \mathbf{W} \mathbf{v}
+
+        \mathbf{W} = \\frac{\mathbf{W}}{\sigma(\mathbf{W})}
+
+
+    Refer to `Spectral Normalization <https://arxiv.org/abs/1802.05957>`_ .
+
+    Args:
+        weight(${weight_type}): ${weight_comment}
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/nn.py]: fill this?

File: python/paddle/fluid/dygraph/nn.py [hunk 3]
```diff
@@ -2333,6 +2486,45 @@ def forward(self, weight):
...
+        Tree-Based Convolution is a kind of convolution based on tree structure.
+        Tree-Based Convolution is a part of Tree-Based Convolution Neural Network(TBCNN),
+        which is used to classify tree structures, such as Abstract Syntax Tree.
+        Tree-Based Convolution proposed a kind of data structure called continuous binary tree,
+        which regards multiway tree as binary tree.
+        The paper of Tree-Based Convolution Operator is here: https://arxiv.org/abs/1409.5718v1
+
+
+        Args:
+            nodes_vector(Variable): input, (Tensor) The feature vector of every node on the tree. The shape of the feature vector must be [max_tree_node_size, feature_size]
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/nn.py]: don't need `input` here

**Review by JiabinYang — COMMENTED** [review]
some comments and plz let @phlrain review this too

File: python/paddle/fluid/dygraph/nn.py (hunk 1 above)
Comment (junjun315) [reply python/paddle/fluid/dygraph/nn.py]: deleted

File: python/paddle/fluid/dygraph/nn.py (hunk 2 above)
Comment (junjun315) [reply python/paddle/fluid/dygraph/nn.py]: deleted

File: python/paddle/fluid/dygraph/nn.py (hunk 3 above)
Comment (junjun315) [reply python/paddle/fluid/dygraph/nn.py]: deleted

**Review by JiabinYang — COMMENTED** [review]
LGTM， plz let @phlrain review this too

File: python/paddle/fluid/dygraph/nn.py [hunk 4]
```diff
@@ -1290,13 +1310,21 @@ class LayerNorm(layers.Layer):
...
-        >>> data = fluid.layers.data(name='data', shape=[3, 32, 32],
-        >>>                          dtype='float32')
-        >>> x = fluid.layers.layer_norm(input=data, begin_norm_axis=1)
+        .. code-block:: python
+
+          import paddle.fluid as fluid
+          import numpy
+
+          with fluid.dygraph.guard():
+              x = numpy.random.random((3, 32, 32)).astype('float32')
```
Comment (phlrain) [inline python/paddle/fluid/dygraph/nn.py:1323]: 这个代码 好像run不了

**Review by phlrain — APPROVED** [review]
LGTM

**Pull request merged**